### PR TITLE
Update Project website and User Guide

### PR DESCRIPTION
**Pull request by tiuweehan**
Update project website: AboutUs, ContactUs, README
 Update User Guide

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -15,7 +11,7 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+* This is a desktop AlgoBase application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
 * It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
```
Comment (le0tan) [inline README.adoc]: Probably we can have some brief introduction to AlgoBase instead?

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -33,9 +33,8 @@ image::Ui.png[width="790"]
 e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 .  Some example commands you can try:
 
-* *`list`* : lists all contacts
-* **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : adds a contact named `John Doe` to the Address Book.
-* **`delete`**`3` : deletes the 3rd contact shown in the current list
+* *`list`* : lists all problems
+* **`add`**`n/factorial m/define a function factorial that takes in a number and n and returns the factorial of the number t/recursion a/Wee Han` : adds a problem called 'factorial' tagged with 'recursion' and authored by 'Wee Han'
```
Comment (le0tan) [inline docs/UserGuide.adoc:37]: There will be more arguments to `add` by the end of v1.1

**Review by le0tan — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 2 above)
Comment (tiuweehan) [reply docs/UserGuide.adoc:37]: What other arguments are there?

File: README.adoc (hunk 1 above)
Comment (tiuweehan) [reply README.adoc]: Ok I'll merge this with the intro for our UG

**Review by le0tan — APPROVED** [review]

**Pull request merged**